jujugui/static/gui/src/app/utils/bundle-importer.js

```diff
@@ -681,7 +681,10 @@ YUI.add('bundle-importer', function(Y) {
         endpoints[index][0] = record[ep[0].replace(/^\$/, '')].get('id');
       }, this);
 
-      const relationId = 'pending-' + record.args[0] + record.args[1];
+      // Use the updated app names here to avoid conflicts if apps with the
+      // same name already exist.
```

Comment:
I don't follow this comment....updated app names from where? Maybe expand on this comment a bit?